### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by soniaroh**

### CODE REVIEW

File: database/schema.sql [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    CREATE DATABASE slideShowData;
+    USE slideShowData;
+    
+    
+        
+    CREATE TABLE products (
+        id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+        productId INT,
+        productName varchar(255),
+        productDescription varchar (255),
```
Comment (michchen58) [inline database/schema.sql:11]: i wouldn't limit this length to 255, maybe use TEXT instead?

**Review by michchen58 — APPROVED** [review]

File: database/schema.sql (hunk 1 above)
Comment (soniaroh) [reply database/schema.sql:11]: good idea! thanks!

File: database/datagenerator.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+var faker = require('faker');
```
Comment (mattminwoolee) [inline database/datagenerator.js:1]: did you set up eslint? Check your json.package and made sure estlint is there and npm i. It'll help clean up your code!

**Review by mattminwoolee — APPROVED** [review]

File: database/datagenerator.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+
+function generateData() {
+    for (var i = 0; i < 100; i++) {
```
Comment (avincenthill) [inline database/datagenerator.js:5]: recommend abstracting 100 as a var for generality

File: server/index.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+require('dotenv').config()
+
+app.use(express.static(__dirname + '/../client/dist'));
+
+
+const PORT = 3000;
+app.listen(PORT, () => {
```
Comment (avincenthill) [inline server/index.js:10]: recommend npm install dotenv to handle environmental vars like PORT as process.env.PORT in a gitignored .env file

**Comment by avincenthill** [comment]
Rec naming PR's in a more specific way

**Comment by avincenthill** [comment]
Keep up the good work Sonia!

**Pull request closed without merging**